pagopa/io-app
# chore(Bonus Pagamenti Digitali): [IAC-7] Remove technical iban feature flag

## Short description
This app remove the technical iban feature flag and the API associated to the old version of the `bpd/io/citizen`.
Note that the name was voluntarily left with the initials "V2" as it reminds us that we are at this version and the endpoint is named like this.

## List of changes proposed in this pull request
- Removed the networking code regarding the `GET bpd/io/citizen`
- Removed the networking code regarding the `PUT bpd/io/citizen`

## How to test
In dev-mode, make sure that when activate the cashback the request are to these endpoint:
- `GET /bonus/bpd/io/citizen/v2`
- `PUT /bonus/bpd/io/citizen/v2`
